src/main/kotlin/org/jitsi/jicofo/conference/source/ConferenceSourceMap.kt

```diff
@@ -43,15 +44,21 @@ data class ConferenceSourceMap(
         }
     )
 
+    /**
+     * An unmodifiable view of this [ConferenceSourceMap].
```

Comment:
The view is unmodifiable, but it's not a COW snapshot of the map, right?

Reply:
Correct. I'll be looking more closely at this, now that I know all the ways it's used (I still need to make changes for thread safety).